docs/storefront/headless/customers.mdx

```diff
@@ -19,15 +19,15 @@ When you sign in a customer using the login mutation, subsequent queries made to
 
 ### Headless and server-side sign in
 
-To make queries from the perspective of a particular customer in headless or server-side code, use [customer impersonation tokens](/docs/start/authentication/graphql-storefront#customer-impersonation-tokens) rather than storefront tokens. 
+To make queries from the perspective of a particular customer in headless or server-side code, use [customer access tokens](/docs/start/authentication/graphql-storefront#customer-access-tokens) rather than storefront tokens. 
 
-Depending on your implementation, you can use the login mutation or a combination of the [Current Customer API](/docs/start/authentication/current-customer), the [Customer Login API](/docs/start/authentication/customer-login), or the REST Management API's [customers feature](/docs/rest-management/customers) to authenticate the customer and get their customer ID. Pass the customer ID with your GraphQL Storefront requests using the `X-Bc-Customer-Id` header to interact with the store from the customer's perspective. The responses will reflect customer-specific pricing, product availability, account information, and more.
+Depending on your implementation, you can use the login mutation, the [Current Customer API](/docs/start/authentication/current-customer), or the REST Management API's [customers feature](/docs/rest-management/customers) to authenticate the customer and get their customer ID. Pass the customer ID with your GraphQL Storefront requests using the `X-Bc-Customer-Access-Token` header to interact with the store from the customer's perspective. The responses will reflect customer-specific pricing, product availability, account information, and more.
 
 ## Customer single sign-on
 
-When a customer signs in to your headless storefront and your application redirects them to a BigCommerce-hosted page, such as checkout, you can use the [Customer Login API](/docs/start/authentication/customer-login) to sign them in to the hosted session. 
+When a customer signs in to your headless storefront and your application redirects them to a BigCommerce-hosted page, such as checkout, you can use a Storefront [token](/docs/rest-authentication/tokens#create-a-token) to sign them in to the hosted session. 
```

Comment:
you just need to use createCartRedirectUrls mutation and use the redirectUrl provided there. It will be session-sync link, that will copy session data from headless storefront to BigCommerce-hosted page.